src/components/chartArea/chartArea.js

```diff
@@ -477,6 +477,10 @@ class ChartArea extends React.Component {
         dataColorStroke.data.stroke = dataSet.stroke;
       }
 
+      if (dataSet.strokeWidth) {
+        dataColorStroke.data.strokeWidth = dataSet.strokeWidth;
+      }
+
```

Comment:
required:
add a minor adjustment to the unit test for `chartArea` so we can see this come through

the `... render basic data` test should do it since it's already using a "strokeWidth" for the threshold
-  https://github.com/RedHatInsights/curiosity-frontend/blob/ci/src/components/chartArea/__tests__/chartArea.test.js#L13-L55

```
...
          stroke: '#lorem',
          strokeWidth: 2
        },
...
          strokeDasharray: '4,3',
          strokeWidth: 3
        }
```